router/internal/edgerouter/config.go

```diff
@@ -14,36 +14,43 @@
 	limitations under the License.
 */
 
-package router
+package edgerouter
 
 import (
-	"errors"
 	"fmt"
 	"github.com/michaelquigley/pfxlog"
 	"github.com/mitchellh/mapstructure"
 	"github.com/openziti/foundation/identity/identity"
 	"github.com/openziti/foundation/transport"
+	"github.com/pkg/errors"
 	"net"
 	"net/url"
 	"strconv"
 	"strings"
+	"time"
 )
 
 const (
-	DefaultHeartbeatIntervalSeconds = 60
-	MinHeartbeatIntervalSeconds     = 10
-	MaxHeartbeatIntervalSeconds     = 60
+	DefaultHeartbeatIntervalSeconds   = 60
+	MinHeartbeatIntervalSeconds       = 10
+	MaxHeartbeatIntervalSeconds       = 60
+	DefaultSessionValidateChunkSize   = 1000
+	DefaultSessionValidateMinInterval = "250ms"
+	DefaultSessionValidateMaxInterval = "1500ms"
```

Comment:
Should include a postfix unit? Example `DefaultSessionValidateMinIntervalMilliseconds. Also not a string? 

We don't follow this pattern anywhere else in the edge right now.

Reply:
We've started using this in the fabric. It's shorter, more flexible and uses the built-in time.ParseDuration method. It's a relatively standard way of specifying intervals/durations. If you're strongly against it, we should talk it through.